internal/gps/cmd_unix.go

```diff
@@ -21,41 +21,33 @@ type cmd struct {
 	ctx context.Context
 	// cancel is called when the graceful shutdown timeout expires.
 	cancel context.CancelFunc
-	cmd    *exec.Cmd
+	Cmd    *exec.Cmd
```

Comment:
Why is this field is being exported?

Reply:
So that `cmd.go` can compile. The windows variant embeds this field directly.